### PR TITLE
Cleanup/sof test

### PR DESCRIPTION
**Pull request by stripes416**
General cleanup of typos, rewording comments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,30 +85,36 @@ apl
...
-* tplgtool.py
-<br> tplgtool dump info from tplg binary file.
+* sof-combinatoric.py
+<br> Used to compute permutations or combinations of the various pipelines
+     avilable during tests if multiple are needed at once.
 
-* sof-tplgreader.py
-<br> tplgtool.py wrapper, it read info from tplgtool.py to analyze tplgs.
+* sof-disk-usage.sh
+<br> Check current disk size for avoid system problem without enough space
```
Comment (aiChaoSONG) [inline README.md]: This line needs some improvement, change to `Used to make sure we have enough disk space to collect logs and avoid system problems`?

File: README.md [hunk 2]
```diff
@@ -21,29 +21,38 @@ sudo apt install expect alsa-utils python3
...
+2020-03-19 22:13:32 UTC [INFO] Catch block option from TPLG_BLOCK_LST will block 'pcm=HDA Digital,Media Playback,DMIC16k' for /lib/firmware/intel/sof-tplg/sof-apl-pcm512x.tplg
+2020-03-19 22:13:32 UTC [INFO] Run command: 'sof-tplgreader.py /lib/firmware/intel/sof-tplg/sof-apl-pcm512x.tplg -f type:playback,both -b pcm:'HDA Digital,Media Playback,DMIC16k' -s 0 -e' to get BASH Array
+2020-03-19 22:13:32 UTC [INFO] Testing: (Round: 1/1) (PCM: Port5 [hw:0,0]<both>) (Loop: 1/3)
+    ...
 ```
-
 ### tools
-call the scripts directly.
--h will show the usage for the tool
+To use tool script, call the scripts directly\n
```
Comment (aiChaoSONG) [inline README.md]: delete '\n'?

File: env-check.sh [hunk 3]
```diff
@@ -89,7 +89,7 @@ case "$SUDO_LEVEL" in
         if [[ "$SUDO_LEVEL" -eq 2 ]]; then
             [[ ! "$SUDO_PASSWD" ]] &&  check_res=1 && out_str=$out_str"\n
 \tPlease setup \e[31mSUDO_PASSWD\e[0min $(dirname $0)/case-lib/config.sh file\n
-\t\tIf you don't want modify this value, you will need to export SUDO_PASSWD\n
+\t\tIf you don't want modify to this value, you will need to export SUDO_PASSWD\n
```
Comment (aiChaoSONG) [inline env-check.sh]: 'want to modify'

File: README.md [hunk 4]
```diff
@@ -21,29 +21,38 @@ sudo apt install expect alsa-utils python3
 sudo adm audio
 
 ### env-check.sh
-You can use this scripts to check what you missed and follow the guide to set the enviroment
+You can use this script to ensure the sof-test enviroment is set up properly
```
[the remark this answers is not in the document]
Comment (stripes416) [reply README.md]: good catch!

File: env-check.sh (hunk 3 above)
Comment (stripes416) [reply env-check.sh]: Actually, this is correct --- it is saying if you dont want to  save the password.

**Comment by Bin-QA** [comment]
@stripes416 please rebase your code

**Comment by stripes416** [comment]
@Bin-QA @wenqingfu I'm not sure what rebasing accomplished here? There wasn't a merge conflict --- it required an approval from @fredoh9 as code owner before it could be merged. It just dismissed the approval reviews from @Bin-QA & @aiChaoSONG.

**Pull request merged**